docs/observability/aws/integrations/amazon-rds.md

```diff
@@ -149,6 +150,18 @@ The Amazon RDS app uses the following logs and metrics:
    "logGroup":"/aws/rds/instance/rds-dbinstance-1/audit"
 }
 ```
+
+```json title ="Authentication Failure (PostgreSQL Log)"
+{"timestamp":1705859815000,
+"message":"2024-01-21 17:56:55 UTC:3.92.54.14(57164):postgresql@postgresql:[43627]:FATAL:  password authentication failed for user \"postgresql\""
```

Comment:
This line ends with two double-quotes. Is that intentional?

Reply:
@kimsauce yes Kim. You can see a  double-quotes opening at the start of value.